.eslintignore

```diff
@@ -14,6 +14,10 @@ htmldomapi.js.map
 is.d.ts
 is.js
 is.js.map
+jsx.d.ts
+jsx-global.d.ts
+jsx.js
+jsx.js.map
```

Comment:
Nit pick: not sorted. But this file is likely going away in #480 so don't bother.